김효중/js/Operator.js

```diff
@@ -0,0 +1,34 @@
+/*
+=를 갖는 dom요소
+*/
+const FindEqualItem = document.querySelector('.item.OperCalculate');
+/*
+수식을 저장할 거임
+*/
+const StoreExpress = document.querySelector('.item.StoreExpresstion');
+/* 
+=가 아닌 item 전부 찾아서 클릭하면 문자열 덧붙임
+*/
+const FindAllItem = document.querySelectorAll('.item:not(.OperCalculate)');
+Array.from(FindAllItem).forEach((EachItem) => {
+  EachItem.addEventListener('click', () => {
+    FindAllItem[0].innerHTML += EachItem.innerHTML;
+  });
+});
```

Comment:
```suggestion
const FindAllItem = document.querySelectorAll('.item:not(.OperCalculate)');
FindAllItem.forEach((EachItem) => {
  EachItem.addEventListener('click', () => {
    FindAllItem[0].innerHTML += EachItem.innerHTML;
  });
});
```

`FindAllItem` 또한 `NodeList` 타입이라 배열 형태로 만들어주지 않아도 `forEach`를 돌릴 수 있을텐데,
배열로 만들어주신 이유가 무엇인가요?

Reply:
```NodeList```타입이 유사 배열이고 ```forEach```메서드를 갖고 있네요. 굳이 배열로 안바꿔도 될 거 같아요!
궁금한 게 있습니다!  유사 배열에서 배열의 메서드를 사용하고 싶으면 반드시 배열로 바꿔야 할까요?.?